recipes-xfce/xfce-nilrt-settings/files/xserver-xfce.default

```diff
@@ -0,0 +1,10 @@
+if [ "$1" = "start" ] && \
+	/usr/local/natinst/bin/nirtcfg --get section=SYSTEMSETTINGS,token=ui.enabled,value=False \
+	| grep -iq false
+then
+	echo "$0: Disabled in system configuration." >&2
+	exit 0
+fi
+
+XUSER=lvuser
+HOME=/home/lvuser
```

Comment:
Seems like this file already exists in [xserver-xfce-init-1.0](https://github.com/ni/meta-nilrt/blob/29fbbd49b52e84de6fb1b3c18702b743d2124420/recipes-xfce/xserver-xfce-init/xserver-xfce-init-1.0/xserver-xfce.conf) and is installed to the same place [here](https://github.com/ni/meta-nilrt/blob/29fbbd49b52e84de6fb1b3c18702b743d2124420/recipes-xfce/xserver-xfce-init/xserver-xfce-init_1.0.bb#L27).